lib/lune.js

```diff
@@ -328,5 +328,39 @@ function phase_hunt (sdate) {
   }
 }
 
+const PHASE_NAMES = ['new', 'q1', 'full', 'q3']
+
+function phase_range (start, end, name) {
+  start = start.getTime()
+  end = end.getTime()
+
+  const phase = PHASE_NAMES.indexOf(name)
+  if (phase === -1) {
+    return null
+  }
+
+  let t = start - 45 * 86400000
+
+  let k
+  {
+    const d = new Date(t)
+    k = Math.floor(12.3685 * (d.getFullYear() + (1.0 / 12.0) * d.getMonth() - 1900))
+  }
+
+  const list = []
+  while (1) {
```

Comment:
Can the condition used in `break` be moved to the while loop condition instead of using `1` here.